spec/models/tax_household_spec.rb

```diff
@@ -137,6 +137,49 @@
     end
   end
 
+  context '#eligible_csr_percent_as_integer' do
```

Comment:
Check code changes for this method. Looks like there could possibly be a bug in finding the product for renewal when CSR kind is csr_limited.